src/lib.rs

```diff
@@ -184,6 +202,15 @@ pub fn sleep(duration: Duration) {
     SpinSleeper::default().sleep(duration);
 }
 
+/// Puts the [current thread to sleep](fn.native_sleep.html) until instant less
+/// the configured native accuracy. Then spins until the specified instant is reached.
+/// 
+/// Convenience function for `SpinSleeper::default().sleep_until(instant)`. Can directly take
+/// the place of `thread::sleep`.
```

Comment:
```suggestion
/// the place of `thread::sleep_until`.
```

Reply:
There's no `thread::sleep_until`, no?

Reply:
There is, though it hasn't stabilized yet. Even unstable it can be directly taken the place of, whereas `thread::sleep` cannot by this.  You could also just remove this sentence.